backend/src/main/java/com/festago/auth/presentation/AuthController.java

```diff
@@ -23,13 +27,16 @@ public AuthController(AuthService authService) {
     }
 
     @PostMapping("/oauth2")
+    @Operation(description = "엑세스 토큰을 기반으로 로그인 요청을 보낸다.")
```

Comment:
"액세스 토큰"은 페스타고에서 발급한 토큰인지, 소셜에서 발급한 토큰인지 헷갈리는 것 같아요!
```suggestion
    @Operation(description = "소셜 액세스 토큰을 기반으로 로그인 요청을 보낸다.")
```
```suggestion
    @Operation(description = "oauth2 액세스 토큰을 기반으로 로그인 요청을 보낸다.")
```
이런 느낌은 어떤가요?!

Reply:
명시적인 것이 좋아 보이네요 ㅎㅎ